docs/bugs.json

```diff
@@ -1,4 +1,14 @@
 [
+    {
+        "uid": "SOL-2021-3",
+        "name": "SignedImmutables",
+        "summary": "Immutable variables of signed integer type shorter than 256 bits can have non-cleaned higher order bits if they are accessed via inline assembly.",
```

Comment:
```suggestion
        "summary": "Immutable variables of signed integer type shorter than 256 bits can have dirty higher order bits if they are accessed via inline assembly.",
```

Reply:
Dirty is maybe the wrong word, because the will always be zero.

Reply:
`Non sign extended` maybe?

Reply:
I used "invalid" now.